content/articles/understanding-different-types-of-menus-in-android-with-kotlin/index.md

```diff
@@ -0,0 +1,261 @@
+### [Android] Understanding Different Types of Menus in Android with Kotlin.
+#### What is a menu as used in android
+Menus are fundamental user interface elements that are frequently used to store actions that would otherwise be buried.
+These hidden commands could be accessible by:
+- Click a button.
+- Using gestures such as long-pressing an image.
+#### Reasons of having menus as a user interface component
+- To free up screen space.
+- To perform operations that are mostly required by users who are not directly represented on the screen.
+- To provide a capability that allows shifting to various parts of the application that are not easily available in the presence of user interface display.
+#### Prerequisites
+- Have android studio installed
+- Have basic knowledge of kotlin programming language.
+- Have a basic knowledge of android.
```

Comment:
Add an intro sentence before listing the prerequisites.
Just an FYI, we call it Android not android. Fix that where applicable 😊